artifact-version-diff/src/main/java/io/quarkus/qe/PrepareOperation.java

```diff
@@ -58,13 +61,21 @@ public static void executeProcess(List<String> command, String errorMsg, Path pa
         try {
             Process process = builder.redirectErrorStream(true)
                     .directory(path.toFile())
-                    .redirectOutput(ProcessBuilder.Redirect.DISCARD)
-                    .redirectError(ProcessBuilder.Redirect.DISCARD)
                     .start();
+            StringBuilder output = new StringBuilder();
+            try (BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()))) {
+                String line;
+                while ((line = reader.readLine()) != null) {
+                    output.append(line).append("\n");
+                }
+            }
             process.waitFor();
+            if(process.exitValue() != 0) {
+                throw new RuntimeException(errorMsg + " Process exited with code: " + process.exitValue() + ". Output: " + output);
+            }
             assertEquals(0, process.exitValue(), errorMsg);
```

Comment:
Shouldn't this be removed as you checknig it above, so it redundant?

Reply:
@jcarranzan thanks for update but atm you don't use process `output` anywhere.  So you just colect it but not print it anywhere. I don't mind have it in if statement as you had before or update assert message. It's up to you what you will choose